friends/src/App.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import './App.css';
+import axios from "axios";
+import FriendsList from './Components/friendsList';
```

Comment:
There were some issues with the paths for files because FriendList was used with capital letter. Same for friends.js.